### PR TITLE
[DOCS-DE] Install Minikube

### PR DESCRIPTION
**Pull request by bene2k1**
German translation of the Install Minikube tutorial

### CODE REVIEW

File: content/de/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+egrep --color 'vmx|svm' /proc/cpuinfo
+```
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Einen Hypervisor installieren
+
+If you do not already have a hypervisor installed, install one for your OS now:
```
Comment (rlenferink) [inline content/de/docs/tasks/tools/install-minikube.md]: This is still in English ;)

**Comment by rlenferink** [comment]
ping @bene2k1

**Comment by bene2k1** [comment]
Done ;-)

**Pull request merged**